### ISSUE TITLE
CI flaky with converge fabric-project

### ISSUE THREAD

**Issue by apeabody** [terraform-google-modules/terraform-google-project-factory#710]
### TL;DR

CI flaky with converge fabric-project

### Observed behavior

vertex "module.fabric-project.module.fabric-project.google_project.project" error: Error enabling the Compute Engine API required to delete the default network: failed to send enable services request: googleapi: Error 403: Service Usage API has not been used in project 84271728655 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/serviceusage.googleapis.com/overview?project=84271728655 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
       Details:
       [
         {
           "@type": "type.googleapis.com/google.rpc.Help",
           "links": [
             {
        "description": "Google developers console API activation",
        "url": "https://console.developers.google.com/apis/api/serviceusage.googleapis.com/overview?project=84271728655"
             }
           ]
         },
         {
           "@type": "type.googleapis.com/google.rpc.ErrorInfo",
           "domain": "googleapis.com",
           "metadata": {
             "consumer": "projects/84271728655",
             "service": "serviceusage.googleapis.com"
           },
           "reason": "SERVICE_DISABLED"
         }
       ]
       , accessNotConfigured

### Terraform Configuration

```hcl
N/A - CI
```

### Terraform Version

```sh
1.1.8
```
